js/util-galery.js

```diff
@@ -0,0 +1,68 @@
+
+const bigPhoto = document.querySelector('.big-picture');
+const body = document.querySelector('.body');
+const commentCounter = document.querySelector('.social__comment-count');
+const commentsLoader = document.querySelector('.comments-loader');
+const closeButton = document.querySelector('.big-picture__cancel  cancel');
```

Comment:
Если несколько классов участвуют в селекторе, то по css (тут прям css селекторы) нужно их писать так:
```suggestion
const closeButton = document.querySelector('.big-picture__cancel.cancel');
```
Т.е. точно так же как если бы ты проставлял стили в css, например:
```css
.big-picture__cancel.cancel {
  background:red;
}
```

Но тут это не нужно и достаточно просто указать 1 класс, он всё равно один в этом контейнере.
Но лучше всегда искать вложенные элементы не с корня (не с document), где это возможно, а от известного родительского элемента, например вот итоговый вариант:
```suggestion
const closeButton = bigPhoto.querySelector('.cancel');
```
Т.е. мы точно уже знаем что эта кнопка внутри нашего bigPhoto поэтому можем поискать её внутри неё, а не обходить всё дерево документа

Reply:
И по аналогии можно так же все внутренние селекты тут заменить на вот такое вот
```ts
const bigPhoto = document.querySelector('.big-picture');
const commentCounter = bigPhoto.querySelector('.social__comment-count');
const commentsLoader = bigPhoto.querySelector('.comments-loader');
const closeButton = bigPhoto.querySelector('.cancel');
const socialComments = bigPhoto.querySelector('.social__comments');
```